### PR TITLE
Refactor konnector action

### PR DESCRIPTION
**Pull request by drazik**
This refactor's goal is to split each subcomponent in its own file, so it's easier to override small parts in customizations. This way we don't have to copy/paste the whole action file to override small UI parts.

### CODE REVIEW

File: src/ducks/transactions/actions/KonnectorAction/BrowserConfigurationModal.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        options={{ slug }}
+        create={client.intents.create.bind(client.intents)}
+      />
+    )
+  }
+}
+
+const BrowserConfigurationModal = withClient(DumbBrowserConfigurationModal)
+
+export default BrowserConfigurationModal
```
Comment (kosssi) [inline src/ducks/transactions/actions/KonnectorAction/BrowserConfigurationModal.jsx]: tu peux mettre directement withClient dans l'export non ?

File: src/ducks/transactions/actions/KonnectorAction/BrowserConfigurationModal.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import IntentModal from 'cozy-ui/react/IntentModal'
+import { withClient } from 'cozy-client'
+
+class DumbBrowserConfigurationModal extends React.Component {
+  render() {
+    const { client, slug, ...rest } = this.props
+
```
Comment (kosssi) [inline src/ducks/transactions/actions/KonnectorAction/BrowserConfigurationModal.jsx]: je mettrai en plus :
```
const options = { slug }
client.intents.create.bind(client.intents)
```

Pour ajouter dans le return juste client.intents.create

File: src/ducks/transactions/actions/KonnectorAction/MobileConfigurationModal.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+    intentWindow.addEventListener('exit', this.props.onComplete)
+  }
+
+  render() {
+    return null
+  }
+}
+
+const MobileConfigurationModal = withClient(DumbMobileConfigurationModal)
```
Comment (kosssi) [inline src/ducks/transactions/actions/KonnectorAction/MobileConfigurationModal.jsx]: Je ne vois pas l'intérêt de mettre cette ligne

File: src/ducks/transactions/actions/KonnectorAction/index.jsx [hunk 4]
```diff
@@ -1,77 +1,25 @@
...
-  title: PropTypes.string.isRequired,
-  description: PropTypes.string.isRequired,
-  caption: PropTypes.string.isRequired,
-  cancelText: PropTypes.string.isRequired,
-  confirmText: PropTypes.string.isRequired
-}
-
 const transactionModalRowStyle = { color: palette.dodgerBlue }
-class _Component extends React.Component {
+class Component extends React.Component {
```
Comment (kosssi) [inline src/ducks/transactions/actions/KonnectorAction/index.jsx]: Component => KonnectorAction ?

File: src/ducks/transactions/actions/KonnectorAction/MobileConfigurationModal.jsx (hunk 3 above)
Comment (drazik) [reply src/ducks/transactions/actions/KonnectorAction/MobileConfigurationModal.jsx]: `DumbMobileConfigurationModal` gets `client` from its props, so we inject it via `withClient`

**Pull request merged**